analysis/estimate/scripts/README.md

```diff
@@ -0,0 +1,8 @@
+This directory contains scripts used in the analysis to predict bulk expression from matched pseudobulk expression.
+
+* `sync-data-files.R`: This script syncs data files for analysis.
+  * It identifies solid tumor samples of interest libraries of interest and obtains their bulk `quant.sf` files and single-cell `_processed.rds` files from S3.
```

Comment:
```suggestion
  * It identifies solid tumor libraries of interest and obtains their bulk `quant.sf` files and single-cell `_processed.rds` files from S3.
```
I think this is what you mean?

Reply:
eh this one is tomato tomAto in terms of sample/library but indeed, only one of those words!